department-of-veterans-affairs/abd-vro
# Move the RRD E2E GH workflow definition

## What was the problem?

* The action had a reference to a non-existent GitHub secret which would already cause a failure on any run.
* It was potentially confusing to have the rrd-end2end-test.yml file in the same directory as all of our other actively used GitHub workflows.

Associated tickets or Slack threads:
- https://github.com/department-of-veterans-affairs/abd-vro/issues/1163

## How does this fix it?[^1]

* Moves the GH workflow definition to its own directory within `domain-rrd/github`
* Adds comment to the dead reference to the GH secret so that if connection would ever be restored it is clear that the secret will need to be recreated

## How to test this PR
- All code changed is not used, only maintained for potential future use cases. 

[^1]: [Pull-Requests guidelines](https://github.com/department-of-veterans-affairs/abd-vro/wiki/Pull-Requests). If PR is significant, update [Current Software State](https://github.com/department-of-veterans-affairs/abd-vro/wiki/Current-Software-State) wiki page.
[^secrel]: To check if a PR will succeed in the SecRel workflow, [test PRs in the SecRel pipeline](https://github.com/department-of-veterans-affairs/abd-vro-internal/wiki/Secure-Release-process#to-test-prs-in-the-secrel-pipeline).
